src/app/drive/types/index.ts

```diff
@@ -145,3 +145,8 @@ export enum DownloadFolderMethod {
   FileSystemAccessAPI = 'file-system-access-api',
   StreamSaver = 'stream-saver',
 }
+
+export enum FreeStoragePlan {
+  simpleName = '2GB',
+  storageLimit = 2147483648,
```

Comment:
Be careful, this could not be the case always, `payments.internxt.com` is the authority here and this app does in fact request plans from there

Reply:
This could be problematic in the future, just to take it for reference.